Tests/conf.json

```diff
@@ -1838,7 +1837,7 @@
             "integrations": "CrowdstrikeFalcon",
             "playbookID": "Test - CrowdStrike Falcon",
             "fromversion": "4.1.0",
-            "timeout": 500,
+            "timeout": 1000,
```

Comment:
@ostolero I increased the timeout since some build machines were failing due to a timeout error.
Example: https://code.pan.run/xsoar/content/-/jobs/33548475#L2743 (Line 2743)